.bazelignore

```diff
@@ -4,3 +4,4 @@ cmake-out/
 cmake-build-debug/
 cmake-build-coverage/
 cmake-build-release/
```

Comment:
I think these can be deleted now. (and from `.dockerignore` and `.gitignore`)

```sh
$ grep "cmake-build" $(git ls-files)
.bazelignore:cmake-build-debug/
.bazelignore:cmake-build-coverage/
.bazelignore:cmake-build-release/
.dockerignore:cmake-build-*/
.gitignore:cmake-build-*/
```

Reply:
No.  CLion creates those, and some of us use CLion.

Reply:
Ack. TIL.